src/txmempool.h

```diff
@@ -21,6 +21,7 @@
 #include <primitives/transaction.h>
 #include <sync.h>
 #include <random.h>
+#include <util/epochguard.h>
```

Comment:
style nit: while touching headers they could be sorted:
```
#include <random.h>
#include <sync.h>
#include <util/epochguard.h>
```

Reply:
Sorted in #20944